gui/layers/_vectors_layer/view/properties.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+# title           : properties.py
+# description     :qt vectors layer
+# author          :bryant.chhun
+# date            :1/16/19
+# version         :0.0
+# usage           :
+# notes           :
+# python_version  :3.6
+
+from PyQt5.QtCore import Qt
+from PyQt5.QtWidgets import QLabel, QComboBox, QSpinBox
+
+from ..._base_layer import QtLayer
+
+
+class QtVectorsLayer(QtLayer):
+
+    def __init__(self, layer):
+        super().__init__(layer)
+
+        # vector color adjustment and widget
+        face_comboBox = QComboBox()
+        colors = self.layer._colors
+        for c in colors:
+            face_comboBox.addItem(c)
+        index = face_comboBox.findText(self.layer.color, Qt.MatchFixedString)
+        if index >= 0:
+            face_comboBox.setCurrentIndex(index)
+        face_comboBox.activated[str].connect(
+            lambda text=face_comboBox: self.changeFaceColor(text))
+        self.grid_layout.addWidget(QLabel('color:'), 3, 0)
+        self.grid_layout.addWidget(face_comboBox, 3, 1)
+
+        # line width in pixels
+        width_field = QSpinBox()
+        value = self.layer.width
+        width_field.setValue(value)
+        width_field.valueChanged.connect(self.changeWidth)
+        self.grid_layout.addWidget(QLabel('width:'), 4, 0)
+        self.grid_layout.addWidget(width_field, 4, 1)
+
+        # averaging combobox
+        averaging_combobox = QComboBox()
+        avg_dims = self.layer._avg_dims
+        for avg in avg_dims:
+            averaging_combobox.addItem(avg)
+        index = averaging_combobox.findText(
+            self.layer.averaging, Qt.MatchFixedString)
+        if index >= 0:
+            averaging_combobox.setCurrentIndex(index)
+            averaging_combobox.activated[str].connect(
+                lambda text=averaging_combobox: self.changeAvgType(text))
+
+        self.grid_layout.addWidget(QLabel('averaging:'), 5, 0)
+        self.grid_layout.addWidget(averaging_combobox, 5, 1)
+
+        # line length
```

Comment:
When I am using `demo_vectors_coord-like` this does not seem to have any effect - why is that? Is that expected?

Reply:
Similar to above, when I use `demo_vectors_image-like` it behaves as expected. If there is a fundamental different between the image like and the coord like we should think about how to best expose this interactivity.